src/DevSoc.Eventathon/Controllers/AttendanceController.cs

```diff
@@ -1,16 +1,30 @@
 ﻿using DevSoc.Eventathon.Calendars.Models;
+using DevSoc.Eventathon.Attendance;
 using Microsoft.AspNetCore.Mvc;
 
 namespace DevSoc.Eventathon.Controllers;
 
 [ApiController]
 public class AttendanceController : ControllerBase
 {
-
+    private readonly IAttendanceService _attendanceService;
+    
+    public AttendanceController(IAttendanceService attendanceService)
+    {
+        _attendanceService = attendanceService;
+    }
+    
     [HttpPost("api/attendance")]
     public Task<IActionResult> ConfirmAttendance([FromBody] AttendanceDefinition definition)
     {
-        // todo: implement attendance
+        _attendanceService.RegisterAttendance(definition);
```

Comment:
`await _attendanceService.RegisterAttendace(definition);`?

Reply:
I realise I forgot to change this in this PR but I have addressed this now